src/v3.rs

```diff
@@ -25,11 +25,14 @@ pub struct V3Sender {
 pub struct SGMailV3 {
     from: Email,
     subject: String,
-    content: Vec<Content>,
     personalizations: Vec<Personalization>,
 
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
Sorry this is a nitpick but could you add a space between fields here? The serde attributes make it a bit noisy.